### PR TITLE
Add connector for Zing MP3

### PR DESCRIPTION
**Pull request by TaQuangKhoi**
Connector for https://zingmp3.vn/

Related to #3404

### CODE REVIEW

**Comment by TaQuangKhoi** [comment]
Hi @yayuyokitano and @inverse , this PR is ready for review.

File: src/connectors/zingmp3.ts [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+const artistSelector = `${playerBar} .is-one-line.is-truncate.subtitle`;
+
+const tractSelector = `${playerBar} .song-title-item`;
+
+Connector.useMediaSessionApi();
+
+Connector.playerSelector = playerBar;
+
+Connector.artistSelector = artistSelector;
```
Comment (inverse) [inline src/connectors/zingmp3.ts:17]: This could be moved inline to simplify things?

File: src/connectors/zingmp3.ts [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+Connector.trackSelector = tractSelector;
```
Comment (inverse) [inline src/connectors/zingmp3.ts:19]: Same for this one

File: src/connectors/zingmp3.ts [hunk 3]
```diff
@@ -0,0 +1,27 @@
+/**
+ * Connector for Zing MP3 - https://zingmp3.vn
```
Comment (inverse) [inline src/connectors/zingmp3.ts:2]: Should we remove this? Not sure what this adds since the site is in the connectors mapping.

**Review by inverse — COMMENTED** [review]
Aside the comments LGTM :+1:

**Comment by TaQuangKhoi** [comment]
@inverse Thanks for your review. I have made changes according to your review.

**Review by inverse — APPROVED** [review]

**Pull request merged**